src/docs/07-operation-guide/05-migration.md

```diff
@@ -0,0 +1,196 @@
+  masterClusterName: "<newClusterName>"
+  currentClusterName: "<newClusterName>"
+  clusterInformation:
+    <currentClusterName>:
+      enabled: true
+      initialFailoverVersion: 1
+      rpcName: "cadence-frontend"
+      rpcAddress: "<currentClusterAddress>"
+    <newClusterName>:
+      enabled: true
+      initialFailoverVersion: 0
+      rpcName: "cadence-frontend"
+      rpcAddress: "<newClusterAddress>"
+```   
+
+Deploy the config.
+In older versions, only `selected-apis-forwarding` is supported. This would require you to deploy a different set of workflow/activity connected to the new Cadence cluster during migration, if high availability/seamless migration is required. Because `selected-apis-forwarding` only forwarding the non-worker APIs.
+
+With `all-domain-apis-forwarding` policy, all worker + non-worker APIs are forwarded by Cadence cluster. You don't need to make any deployment change to your workflow/activity workers during migration. Once migration, let all workers connect to the new Cadence cluster before removing/shutdown the old cluster.
+
+Therefore, it's recommended to upgrade your Cadence cluster to a higher version with `all-domain-apis-forwarding` policy supported. The below steps assuming you are using this policy.
+
+
+### Step 2 - Test Replicating one domain
+
+First of all, try replicating a single domain to make sure everything work. Here uses `domain update` to failover, you can also use `managed failover` feature to failover. You may use some testing domains for this like `cadence-canary`.
+
+* 2.1 Assuming the domain only contain `currentCluster` in the cluster list, let's add the new cluster to the domain.
+```bash
+cadence --address <currentClusterAddress> --do <domain_name> domain update --clusters <currentClusterName> <newClusterName>
+```
+
+Run the command below to refresh the domain after adding a new cluster to the cluster list; we need to update the active_cluster to the same value that it appears to be.
+
+```
+cadence --address <currentClusterAddress> --do <domain_name> domain update --active_cluster <currentClusterName>
+```
+
+
+* 2.2 failover the domain to be active in new cluster
+```
+cadence --address <currentClusterAddress> --do workflow-prototype domain update --active_cluster <newClusterName>
+```
+
+Use the domain describe command to verify the entire domain is replicated to the new cluster.
+
+```
+cadence --address <newClusterAddress> --do <domain_name> domain describe
+```
+Find an open workflowID that we want to replicate (you can get it from the UI). Use this command to describe it to make sure it’s open and running:
+
+```
+cadence --address <initialClusterAddress> --do <domain_name> workflow describe --workflow_id <wfID>
+```
+Run a signal command against any workflow and check that it was replicated to the new cluster. Example:
+
+```
+cadence --address <initialClusterAddress> --do <domain_name> workflow signal --workflow_id <wfID> --name <anything, e.g. xdcTest>
```

Comment:
could we mention the signal is to trigger the replication use a fake signal which will not impact the workflow?